go.mod

```diff
@@ -67,9 +67,9 @@ require (
 	github.com/coreos/go-semver v0.3.0 // indirect
 	github.com/coreos/go-systemd v0.0.0-20191104093116-d3cd4ed1dbcf // indirect
 	github.com/coreos/go-systemd/v22 v22.3.2 // indirect
-	github.com/evanphx/json-patch v4.12.0+incompatible
+	github.com/evanphx/json-patch v5.6.0+incompatible
```

Comment:
That one is probably a false positive based on the way you bumped. 

We already use: `github.com/evanphx/json-patch/v5 v5.6.0`

Can you try to use the v5 import path in cluster/topology.go? This way we should be able to get rid of the v4 dependency entirely